runsc/boot/loader.go

```diff
@@ -290,6 +290,8 @@ func New(args Args) (*Loader, error) {
 		log.Infof("Setting total memory to %.2f GB", float64(args.TotalMem)/(1<<30))
 	}
 
+	runtime.GOMAXPROCS(args.NumCPU)
```

Comment:
Nit: please move this to the line right after (or before) `log.Infof("CPUs: %d",...`.

Reply:
yep!